docs/hitran/hitran.rst

```diff
@@ -0,0 +1,37 @@
+.. doctest-skip-all
+
+************************************
+HITRAN Queries (`astroquery.hitran`)
+************************************
+
+Getting started
+===============
+
+This module provides an interface to the `HITRAN`_ database API.  It can
+download a data file including transitions for a particular molecule in a given
+wavenumber range.  The file is downloaded in the `~hitran.cache_location`
+directory and can be opened with a reader function that returns a table of
+spectral lines including all accessible parameters.
+
+Examples
+========
+
+This will download all transitions of the main isotopologue of water between
+the wavenumbers of 3400 and 4100 cm\ :sup:`-1`\ .
```

Comment:
Does this work for you locally? Singe backticks are special for sphinx, probably you will need to use double backticks here